Jenkinsfile.iotech

```diff
@@ -0,0 +1,27 @@
+pipeline {
+    agent { label "docker-${params.ARCH}" }
+    options { timestamps() }
+    stages {
+        stage('Run test') {
+            steps {
+                sh "docker pull iotech-releases.jfrog.io/edgex-taf_x86_64:1.0.0"
+                sh "docker run --rm --network host -v ${env.WORKSPACE}:${env.WORKSPACE} -w ${env.WORKSPACE} -v /var/run/docker.sock:/var/run/docker.sock iotech-releases.jfrog.io/edgex-taf_x86_64:1.0.0 --exclude Skipped -u . -p ${params.PROFILE}"
```

Comment:
You can make use of methods of Jenkins to play around with docker like `.inside`
Also, not sure why the file name is `.iotech`

Reply:
Sorry, we use this file to run the test job on our Jenkins server. I will create another PR to run test on the community once [edgex-taf-comm PR merged](https://github.com/edgexfoundry/edgex-taf-common/pull/7).